src/accessibility.cs

```diff
@@ -53,4 +140,173 @@ interface AXCustomContentProvider
+		AXDataPointValue ValueWithNumber (double number);
+
+		[Static]
+		[Export ("valueWithCategory:")]
+		AXDataPointValue ValueWithCategory (string category);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXDataSeriesDescriptor : NSCopying
+	{
+		[NullAllowed, Export ("name")]
+		string Name { get; set; }
+
+		[Export ("attributedName", ArgumentSemantic.Copy)]
+		NSAttributedString AttributedName { get; set; }
+
+		[Export ("isContinuous")]
+		bool IsContinuous { get; set; }
+
+		[Export ("dataPoints", ArgumentSemantic.Copy)]
+		AXDataPoint[] DataPoints { get; set; }
+
+		[Export ("initWithName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string name, bool isContinuous, AXDataPoint[] dataPoints);
+
+		[Export ("initWithAttributedName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAttributedString attributedName, bool isContinuous, AXDataPoint[] dataPoints);
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Flags]
+	[Native]
+	public enum AXHearingDeviceEar : ulong
+	{
+		None = 0,
+		Left = 1 << 1,
+		Right = 1 << 2,
+		Both = Left | Right,
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Static]
+	[Partial]
+	partial interface AXHearingUtilities
+	{
+		[Field ("AXMFiHearingDeviceStreamingEarDidChangeNotification")]
+		NSString StreamingEarDidChangeNotification { get; }
+
+		[Field ("AXMFiHearingDevicePairedUUIDsDidChangeNotification")]
+		NSString PairedUUIDsDidChangeNotification { get; }
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface AXLiveAudioGraph
+	{
+		[Static]
+		[Export ("start")]
+		void Start ();
+
+		[Static]
+		[Export ("updateValue:")]
+		void UpdateValue (double value);
+
+		[Static]
+		[Export ("stop")]
+		void Stop ();
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[Native]
+	public enum AXNumericDataAxisDescriptorScale : long
+	{
+		linear = 0,
+		log10,
+		ln,
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXNumericDataAxisDescriptor : AXDataAxisDescriptor
+	{
+		[Export ("scaleType", ArgumentSemantic.Assign)]
+		AXNumericDataAxisDescriptorScale ScaleType { get; set; }
+
+		[Export ("lowerBound")]
+		double LowerBound { get; set; }
+
+		[Export ("upperBound")]
+		double UpperBound { get; set; }
+
+		[Export ("valueDescriptionProvider", ArgumentSemantic.Copy)]
+		Func<double, NSString> ValueDescriptionProvider { get; set; }
+
+		[Export ("gridlinePositions", ArgumentSemantic.Copy)]
+		NSNumber[] GridlinePositions { get; set; }
+
+		[Export ("initWithTitle:lowerBound:upperBound:gridlinePositions:valueDescriptionProvider:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string title, double lowerbound, double upperBound, [NullAllowed] NSNumber[] gridlinePositions, Func<double, NSString> valueDescriptionProvider);
```

Comment:
camel-case `lowerBound` to match your `upperBound`

Reply:
good catch, thank you!